ansible_rulebook/websocket.py

```diff
@@ -104,22 +104,26 @@ async def send_event_log_to_websocket(
 
             while True:
                 event = await event_log.get()
+                logger.debug(f"Event received, {event}")
                 await websocket.send(json.dumps(event))
 
-                if event == dict(type="Shutdown"):
+                if event == dict(type="Exit"):
```

Comment:
@mkanoor why do we change type from Shutdown to Exit? Server side has a list of acceptable types. We need add `Exit` into it.
https://github.com/ansible/eda-server/blob/main/src/aap_eda/wsapi/consumers.py#L34

Reply:
@hsong-rh That is an internal message asking the feedback task to end, it will not be sent to the server. I have fixed it to not send the Exit message.